### ISSUE TITLE
Doesn't seem to load selected preset at boot, until I move the brightness slider.

### ISSUE THREAD

**Issue by eborchardt** [Aircoookie/WLED#2521]
### What happened?

I created a preset that is just a dim red color. It works when I select the preset, the lights change to dim red. I go to Config | LED Preferences | Defaults and select "Apply preset [1] at boot". Then I save and reboot the ESP8622. The preset is never loaded and, furthermore, it won't change to the preset when I click the preset button, unless I first adjust the brightness arbitrarily. 

### To Reproduce Bug

See above...

### Expected Behavior

I expected the preset to be loaded on boot.

### Install Method

Binary from WLED.me

### What version of WLED?

0.13.0-b6 (build 2112080)

### Which microcontroller/board are you seeing the problem on?

ESP8266

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blazoncek** [comment]
Post a preset JSON API.

**Comment by eborchardt** [comment]
Here is my Dim Red json:
`{"on":true,"bri":47,"transition":7,"mainseg":0,"seg":[{"id":0,"start":0,"stop":1,"grp":1,"spc":0,"of":0,"on":true,"bri":255,"cct":127,"col":[[255,0,0],[8,255,0],[0,0,255]],"fx":0,"sx":128,"ix":128,"pal":0,"sel":true,"rev":false,"mi":false},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0},{"stop":0}]}`

**Comment by blazoncek** [comment]
What is your boot setting in LED settings?
Post screenshot?

The preset and state all look ok.

**Comment by eborchardt** [comment]
![image](https://user-images.githubusercontent.com/56270998/151846731-96f7ff17-19df-4162-a02a-51a13f0a0952.png)

**Comment by eborchardt** [comment]
Actually, I ended up taking it apart this morning to see if I got the same result with an ESP32 and it worked fine with the ESP32. I just swapped the ESP8266 back in and now the issue is no longer occurring. I'm not sure how or why, but it seems there must have been a bad connection or something. It is puzzling how a bad connection could have caused it to exhibit the behavior I was experiencing, but it seems it must have been the case.
